research/evm_explorer_comparison.md

```diff
@@ -0,0 +1,152 @@
+# A Comprehensive Analysis of Etherscan and Blockscout: Pros, Cons, and Alternatives
+
+## Introduction
+
+As a developer building an OP-Stack-based rollup, one of the critical decisions you'll need to make is selecting a suitable blockchain explorer. Blockchain explorers are like a pair of binoculars - providing transparency and insights into what is happening on the chain. They function as block explorer search engines to offer users an interface that provides them with detailed information on transactions, addresses, contracts and network statistics. The blockchain explorer can be used to verify the status of transactions, look at all things within each block and gets a better understanding of how the chain works.
```

Comment:
Instead of `They function as block explorer search engines to offer users an interface that provides them with detailed information on transactions, addresses, contracts and network statistics.` we should say something like `They function as blockchain search engines to offer users an interface that provides them with detailed information on transactions, addresses, contracts and network statistics.`